components/collapse/demo/custom.ts

```diff
@@ -4,10 +4,13 @@ import { Component } from '@angular/core';
   selector: 'nz-demo-collapse-custom',
   template: `
     <nz-collapse [nzBordered]="false">
-      <nz-collapse-panel *ngFor="let panel of panels" [nzHeader]="panel.name" [nzActive]="panel.active"
-        [ngStyle]="panel.customStyle">
+      <nz-collapse-panel *ngFor="let panel of panels; let isLast = last" [nzHeader]="panel.name" [nzActive]="panel.active"
```

Comment:
Wow. Actually I learned something here! 😆

Reply:
😆 Now I've changed it to `first`